app/controllers/WrongWithMovementController.scala

```diff
@@ -43,17 +43,91 @@ class WrongWithMovementController @Inject()(
+
+          val newUserAnswers = allOptionsNotChecked.foldLeft(request.userAnswers) {
+            case (answers, WrongWithMovement.Shortage) =>
+              answers
+                .remove(AddShortageInformationPage)
+                .remove(ShortageInformationPage)
+            case (answers, WrongWithMovement.Excess) =>
+              answers
+                .remove(AddExcessInformationPage)
+                .remove(ExcessInformationPage)
+            case (answers, WrongWithMovement.Damaged) =>
+              answers
+                .remove(AddDamageInformationPage)
+                .remove(DamageInformationPage)
+            case (answers, WrongWithMovement.BrokenSeals) =>
+              answers
+                .remove(AddSealsInformationPage)
+                .remove(SealsInformationPage)
+            case (answers, WrongWithMovement.Other) =>
+              answers
+                .remove(OtherInformationPage)
+            case (answers, _) => answers
+          }
+
+          saveAndRedirect(WrongWithMovementPage, values, newUserAnswers, mode)
+        }
+      )
+    }
+  }
+
+  def loadWrongWithItem(ern: String, arc: String, idx: Int, mode: Mode): Action[AnyContent] =
+    onPageLoad(WrongWithItemPage(idx), ern, arc, routes.WrongWithMovementController.submitWrongWithItem(ern, arc, idx, mode))
+
+  def submitWrongWithItem(ern: String, arc: String, idx: Int, mode: Mode): Action[AnyContent] = {
+    authorisedDataRequestAsync(ern, arc) { implicit request =>
+      formProvider(WrongWithItemPage(idx)).bindFromRequest().fold(
+        formWithErrors =>
+          Future.successful(BadRequest(view(
+            WrongWithItemPage(idx),
+            formWithErrors,
+            routes.WrongWithMovementController.submitWrongWithItem(ern, arc, idx, mode)
+          ))),
+        (values: Set[WrongWithMovement]) => {
+
+          val allOptionsNotChecked: Seq[WrongWithMovement] = WrongWithMovement.individualItemValues.filterNot(values.contains)
+
+          val newUserAnswers = allOptionsNotChecked.foldLeft(request.userAnswers) {
+            case (answers, WrongWithMovement.ShortageOrExcess) =>
+              answers
+                .remove(ItemShortageOrExcessPage(idx))
+            case (answers, WrongWithMovement.Damaged) =>
+              answers
+                .remove(AddItemDamageInformationPage(idx))
+                .remove(ItemDamageInformationPage(idx))
+            case (answers, WrongWithMovement.BrokenSeals) =>
+              answers
+                .remove(AddItemSealsInformationPage(idx))
+                .remove(ItemSealsInformationPage(idx))
+            case (answers, WrongWithMovement.Other) =>
+              answers
+                .remove(ItemOtherInformationPage(idx))
+            case (answers, _) => answers
+          }
+          saveAndRedirect(WrongWithItemPage(idx), values, newUserAnswers, mode)
+        }
+      )
```

Comment:
Should only do this if the set of selected checkboxes is different to the previous set of selected checkboxes. The back and then forward without changing scenario again.